### ISSUE TITLE
Split training mission in 2 brances?

### ISSUE THREAD

**Issue by 132ndNeck** [132nd-vWing/132nd-Virtual-Wing-Training-Mission-Tblisi#27]
I suggest that we split the training mission in 2 branches?

Branch 1: MP only for dedicated MP training events. This is the branch to be flown on training events.
Branch 2: For use in singleplayer for various training (Formation trainer, SAM's etc). (Can of course be used in MP by several people when two pilots decide to fly together).

**Comment by theendsofinvention** [comment]
I don't know about this one.

I fail to see the added value. Maintaining the two (potentially conflicting) branches side by side sounds like a lot of overhead for little added value.

Instead, several options may be made optional or be turned on and off via F10 menu. Possibly, access to these options may be restricted.

**Comment by 132nd-Entropy** [comment]
Im against that, simply because I like to have options. Why should some one during a MP training not be able to practice SAM evasion and at the same time enjoy the MP environment and ATC? If the SAMs are in the way, they can be moved to another area. If they are in the way and cannot be moved because they are in the way no matter where they are, then I guess they need to be taken out If the Formation trainer is a bad idea, because it interferes with ATC etc, whihc I totally understand,  it can be taken out (was there before I started working on it). 

Im all for deleting stuff when it disturbs operations, but Im at the same time for keeping stuff that is cool  as long as they are not in the way. 

In any case I think maintain two branches of the same mission is creating more work and divides resources.

**Comment by 132ndNeck** [comment]
Copy.

**Comment by 132nd-etcher** [comment]
@132ndNeck Do you have something specific in mind that you'd like moved apart from the regular TRMT ? Or maybe even a list of things ?

**Comment by 132ndNeck** [comment]
@132nd-etcher No, I have not gone into the details of it. It is just a general observation from my side. 
I created this training mission to mimic real life training (with the adaption to DCS limitations and making it feasible of course).
I my perspective the training mission is a safe environment where one can train without getting shot.

And while I have been in the US and now on deployment, I have a feel that the training mission may be slowly moving away from what I envisioned with making the mission, with addition of red forces from time-to-time.

I just want to make sure that the training mission continue to be what I envision for a training mission.
I do not have a concern with the training mission as it is now, and this was just as suggestion by me to make sure to avoid getting in that position. I dont want to end in a position where someone puts down a lot of effort in something, and then I, or someone else is not in favor of doing including it in the mission. So I think using these issues to take up stuff for the training mission is a good way of making sure we are all on the same page for the development of this mission so that we dont step on any toes.

There has been done some amazing work on the training mission, and the current functionality is way better than what started out as this training mission by my way back.
